### PR TITLE
MSAID output compatibility

### PR DESCRIPTION
**Pull request by rodvrees**
Includes a Jupyter Notebook to prepare MSAID output: protein information is not added to precursor information, only an ID. The Jupyter Notebook adds protein information based on the protein ids from proteingroups.tsv

Also converts extracted param: 'quantification_method_DIANN' into 'quantification_method'. Was originally meant specifically for different quant methods implemented in DIA-NN, but MSAID also has a bunch of quantification options, so makes sense to generalize the param. 

Also includes a fix that allows output of AlphaDIA two-step search, where it can happen that species information from fasta gets lost

### CODE REVIEW

**Comment by RobbinBouwmeester** [comment]
@rodvrees ready to merge?

**Comment by rodvrees** [comment]
@RobbinBouwmeester Yes, I'm ready!

File: proteobench/datapoint/quant_datapoint.py [hunk 1]
```diff
@@ -53,6 +53,7 @@ class Datapoint:
     intermediate_hash: str = ""
     results: dict = None
     median_abs_epsilon: int = 0
+    # TODO nr_prec doesnt get updated in final df, always 0
```
Comment (RobbinBouwmeester) [inline proteobench/datapoint/quant_datapoint.py:56]: Remove this comment? Is resolved?

File: test/params/Proteobench_default_actual_details.csv
Comment (RobbinBouwmeester) [inline test/params/Proteobench_default_actual_details.csv:1]: Could we give this a name that makes more sense? E.g., add MSAID?

File: test/params/Proteobench_default_actual_details.tsv
Comment (RobbinBouwmeester) [inline test/params/Proteobench_default_actual_details.tsv:1]: Could we give this a name that makes more sense? E.g., add MSAID?

**Review by RobbinBouwmeester — CHANGES_REQUESTED** [review]

File: test/params/Proteobench_default_actual_details.tsv
Comment (rodvrees) [reply test/params/Proteobench_default_actual_details.tsv:1]: Done

**Review by RobbinBouwmeester — APPROVED** [review]

**Pull request merged**